app/files/file_utils.py

```diff
@@ -13,29 +16,49 @@ def job_file_name_for_job(job_id):
     return "{}-dvla-job.text".format(job_id)
 
 
+def job_id_from_filename(job_filename):
+    return job_filename.replace("-dvla-job.text", "")
+
+
 def get_file_from_s3(bucket_name, job_id):
-    s3 = boto3.client('s3')
     filename = job_file_name_for_job(job_id)
-    with open("{}/{}".format(current_app.config['LOCAL_FILE_STORAGE_PATH'], filename), 'wb') as job_file:
-        s3.download_fileobj(bucket_name, filename, job_file)
+    try:
+        s3 = boto3.client('s3')
+        with open(full_path_to_file(filename), 'wb+') as job_file:
+            s3.download_fileobj(bucket_name, filename, job_file)
+        return True
+    except Exception as e:
+        os.remove(full_path_to_file(filename))
+        current_app.logger.error(e)
+        current_app.logger.error("Failed to download {}".format(filename))
```

Comment:
`current_app.logger.exception("Failed to download{}".format(filename))` can replace this and the line above - `.exception` looks in the current stack for an exception object, then nicely prints the stack trace (whereas `logger.error(e)` will just print a repr of the exception obj